README.md

```diff
@@ -21,26 +21,6 @@ This is a Ruby on Rails application.
 - [yarn][] - package manager for JavaScripts
 - [imagemagick][] - image manipulation library
 
-### Running the application
-
-The first time you run this application you'll want to set you
```

Comment:
I suspect this line is still relevant to new people using the app.

Reply:
I kept it. I took the opportunity to rearrange some sections, too - `permissions` didn't need to be linked to.